Dockerfile.build

```diff
@@ -36,6 +36,22 @@ RUN npm install
 RUN npm run build -- --config grpc/src/main/resources/assets/pt/webpack.config.js
 
 
+# Download + set up files needed for tests
+FROM gcr.io/google.com/cloudsdktool/cloud-sdk:latest as test-build
+
+ARG MININORCAL_OSM_PATH
+ARG MININORCAL_GTFS_PATH
+ARG GOOGLE_CREDENTIALS_FILE
+RUN echo $GOOGLE_CREDENTIALS_FILE
+
+
+WORKDIR /graphhopper
+COPY . .
+RUN ls -la
```

Comment:
I'm assuming this is just for debug?

Reply:
Yup 😀 https://github.com/replicahq/graphhopper/pull/81#discussion_r683875155